src/main/java/reactor/kafka/receiver/KafkaReceiver.java

```diff
@@ -46,10 +46,24 @@
      *        must be set on the options instance prior to creating this receiver.
      * @return new receiver instance
      */
-    public static <K, V> KafkaReceiver<K, V> create(ReceiverOptions<K, V> options) {
+    static <K, V> KafkaReceiver<K, V> create(ReceiverOptions<K, V> options) {
```

Comment:
Why did we change that to package-private? should not it be public?

Reply:
they are by default public. so didn't change them to package-private